src/opensles/AudioOutputStreamOpenSLES.cpp

```diff
@@ -110,6 +110,34 @@ SLuint32 AudioOutputStreamOpenSLES::channelCountToChannelMask(int channelCount)
 }
 
 Result AudioOutputStreamOpenSLES::open() {
+    // Log unsupported attributes
+    // only report if changed from the default
+    // Usage::Media
+    // NOT TRUE! See OpenSLES_convertOutputUsage
```

Comment:
What is not true?

Reply:
That statement (or implication) that mUseage is not supported. I just left it in temporarily until I was sure I was doing the right thing.